pyartcd/pyartcd/pipelines/update_golang.py

```diff
@@ -251,6 +238,42 @@ async def run(self):
             dry_run=self.dry_run,
         )
 
+    def regen_repo_with_complete(self, el_v, timeout):
+        build_tag = f'rhaos-{self.ocp_version}-rhel-{el_v}-build'
+        task_id = self.koji_session.newRepo(build_tag)
+        _LOGGER.info(f"Regenerating repo for tag: {build_tag} with task {task_id}, waiting max to 10 mins...")
+        # regen repo usually take a few mins, wait max to 10 mins
+        wait_min = 0
+        while wait_min < timeout:
+            time.sleep(60)
+            gen_repo_task_info = self.koji_session.getTaskInfo(task_id)
+            if gen_repo_task_info['completion_time']:
+                _LOGGER.info(f"Regenerating repo for tag: {build_tag} complete.")
+                return True
+            _LOGGER.info(f"Regenerating repo for tag: {build_tag} task {task_id} is still running")
+            wait_min += 1
+        return False
+
+    def tag_builds_override(self, el_v, nvr):
```

Comment:
`tag_builds` here is sufficient imo, `override` doesn't add to the meaning

Reply:
updated